### PR TITLE
playbook to install a deb package

### PR DESCRIPTION
**Pull request by scoop206**
This playbook will install a package to target hosts.
It's immediate use will be for updating the kernel package.

### CODE REVIEW

File: playbooks/install_deb_package.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# simple playbook to install an deb package w/ apt
+# USAGE: pass an extra var w/ your ansible command
+# indicating the package.
+#
+# For example, this will install the tmux packaage: 
+# ... -e package=tmux
+
+- name: install a deb package
+  hosts: all
```
Comment (nirajdp76) [inline playbooks/install_deb_package.yml]: shouldn't this be `hosts: all:!vyatta-*`

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by twaldrop** [comment]
retest

**Pull request merged**